airbyte-commons-server/src/main/java/io/airbyte/commons/server/handlers/SchedulerHandler.java

```diff
@@ -417,12 +438,27 @@ private void generateCatalogDiffsAndDisableConnectionsIfNeeded(final SourceDisco
       }
       updateObject.status(connectionStatus);
       connectionsHandler.updateConnection(updateObject);
+      if (shouldNotifySchemaChange(diff, connectionRead, discoverSchemaRequestBody)) {
+        final String url = webUrlHelper.getConnectionUrl(workspaceId, connectionRead.getConnectionId());
+        try {
+          notificationWorkflow.sendSchemaChangeNotification(connectionRead.getConnectionId(), url);
```

Comment:
Shouldn't the notification also take the information from the update that could happen line 450?

Reply:
which information are you thinking? I was hoping to add the diff info afterward, but right now the notification includes whether or not it's a breaking change and a link to the connection page

Reply:
Yes, breaking change is included in the notification - diff isn't yet